code.js

```diff
@@ -1,3 +1,42 @@
-function divideAndConquerSum(a) {
-    return a;
+function divideAndConquerSum(array) {
+    let sum = 0;
+    switch(array.length){
+        case 0:
+            return 0;
+            break
+        case 1:
+            return array[0];
+            break;
+        case 2:
+            return array[0] + array[1];
+            break;
+        default:
+            sum = helper(array);
+            return sum;
+    }  
+}
+
+function helper(arr){
+    switch(arr.length){
+        case 0:
```

Comment:
Do you need these checks here if you're doing them in the above function as well?

Reply:
I suppose not. I should have it fixed now